benches/tanimoto_bench.rs

```diff
@@ -1,19 +1,23 @@
-// #![allow(soft_unstable)]
-// #![feature(test)]
-//
-// extern crate test;
-// use test::Bencher;
-//
-// #[bench]
-// fn bench_tanimoto_similarity(b: &mut Bencher) {
-//     let smiles1 = "c1ccccc1CCCCCCCC";
-//     let mol1 = Molecule::new(smiles1, "").unwrap();
-//
-//     let smiles2 = "c1ccccc1CCCCCC";
-//     let mol2 = Molecule::new(smiles2, "").unwrap();
-// 
-//     b.iter(|| mol1.get_tanimoto_similarity(&mol2, "", ""));
-// }
+#![feature(test)]
+
+use rdkit::*;
+use cheminee::analysis::compound_processing::*;
+
+extern crate test;
+use test::Bencher;
+
+#[bench]
+fn bench_tanimoto_distance(b: &mut Bencher) {
+    let smiles1 = "[N]Cc1cncc2c(=O)c3cccc(CCC(=O)O)c3[nH]c12";
+    let (_proc_smiles1, fingerprint1, _descriptors1) = process_cpd(smiles1).unwrap();
+
+    let smiles2 = "CCc1cccc2c(=O)c3cncc(CN)c3[nH]c12";
+    let (_proc_smiles2, fingerprint2, _descriptors2) = process_cpd(smiles2).unwrap();
+
+
+    b.iter(|| fingerprint1.tanimoto_distance(&fingerprint2));
+}
```

Comment:
how about benching the fingerprint generation as well?

Reply:
Done!